XAM/Models/Exam.cs

```diff
@@ -10,9 +10,21 @@ public Exam(string name, DateTime date)
         Date = date;
         Flashcards = new List<Flashcard>(); // Initialize the list of flashcards
     }
-}
 
-public struct Flashcard
+    // Method to remove a flashcard
+    public bool RemoveFlashcard(string frontText)
+    {
+        var flashcardToRemove = Flashcards.FirstOrDefault(f => f.FrontText == frontText);
+        if (!flashcardToRemove.Equals(default(Flashcard))) // Check if flashcard was found
+        {
+            Flashcards.Remove(flashcardToRemove);
+            return true; // Successfully removed
+        }
+        return false; // Not found
+    }
+
+
+    public struct Flashcard
```

Comment:
I believe, that struct flashcard should be moved to the outside of exam class parenthesis, as you can't access it from HomeController, and, at least for me, I get an error that Flashcard isn't recognized.  RemoveFlashcard function on the other hand should stay in the Exam class.

Reply:
I changed my code so the struct would be as it was because I thought it would be easier for everyone. I also updated the frontend because last time the button was behind the flashcard.